client/src/app/plants/plant.component.ts

```diff
@@ -1,19 +1,24 @@
-import { Component, OnInit } from '@angular/core';
+
 import { PlantListService } from "./plant-list.service";
 import { Plant } from "./plant";
+import { Component, OnInit, Input, Output, OnChanges, EventEmitter, trigger, state, style, animate, transition } from '@angular/core';
```

Comment:
That's probably correct, they probably aren't all required. I believe I copied those from your Iteration 1 repo but I didn't copy all of the functions so not all of the imports are used.